pkg/backend/apply.go

```diff
@@ -122,9 +123,9 @@ func PreviewThenPrompt(ctx context.Context, kind apitype.UpdateKind, stack Stack
 	}
 
 	// Otherwise, ensure the user wants to proceed.
-	err = confirmBeforeUpdating(kind, stack, events, op.Opts)
+	err := confirmBeforeUpdating(kind, stack, events, op.Opts)
 	close(eventsChannel)
-	return changes, err
+	return changes, result.WrapIfNonNil(err)
```

Comment:
new helper i added.  If you give it a nil-error, you get a nil-result **not** a result which wraps that nil-error.  This should be used if you're not directly inside an `err != nil` check.